Tracking/src/ClicEfficiencyCalculator.cc

```diff
@@ -613,7 +613,7 @@ void ClicEfficiencyCalculator::processEvent( LCEvent* evt ) {
     nReconstructable++;
     m_thetaPtMCParticle->Fill(mcTheta,mcPt);
     std::vector<TrackerHit*> trackHits = particleHits[particle];
-    int uniqueHits = getUniqueHits(trackHits,m_encoder);
+    //int uniqueHits = getUniqueHits(trackHits,m_encoder);
```

Comment:
@eleogran maybe better removing it? Or do we want to introduce a variable for this in the TTree?